x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/edit_policy.helpers.tsx

```diff
@@ -21,10 +21,12 @@ import { KibanaContextProvider } from '../../../public/shared_imports';
 import { AppServicesContext } from '../../../public/types';
 import { createBreadcrumbsMock } from '../../../public/application/services/breadcrumbs.mock';
 
+import { TestSubjects } from '../helpers';
+import { POLICY_NAME } from './constants';
+
 type Phases = keyof PolicyPhases;
 
-import { POLICY_NAME } from './constants';
-import { TestSubjects } from '../helpers';
+window.scrollTo = jest.fn();
```

Comment:
This should no longer be necessary, if it is then we probably need to remove this from the code because AFAIK this is used to scroll to the first visible error on form submission. It's nice behaviour, but has not been mapped onto the new UI with collapsible settings in phases.

Reply:
We still use a scroll to 0,0 for when the policy has loaded ([edit_policy.tsx#L66](https://github.com/elastic/kibana/blob/bf7fdfc87cb04ecb5f6081d9056102d707997e9b/x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.tsx#L66)). It doesn't affect the tests results though, but I prefer to keep it there for now to avoid unnecessary errors in output.